cuenca_validations/validators.py

```diff
@@ -1,12 +1,23 @@
-__all__ = ['sanitize_dict']
-
 import datetime as dt
 from enum import Enum
+from typing import Any
+
+__all__ = ['sanitize_dict']
 
 
 def sanitize_dict(d: dict):
     for k, v in d.items():
-        if isinstance(v, dt.date):
-            d[k] = v.isoformat()
-        elif isinstance(v, Enum):
-            d[k] = v.value
+        try:
+            d[k] = sanitize_item(v)
+        except AttributeError:
+            ...
+
+
+def sanitize_item(item: Any) -> Any:
+    if isinstance(item, dt.date):
+        result = item.isoformat() + 'Z'  # Siempre usamos UTC
```

Comment:
We should stick to English or Spanish vs mixing. I chose English for this library since it's used by `cuenca-python`. 

For the comment, simply state "comply with iso8601". Python annoyingly violates the standard